### PR TITLE
rudimentary missings support

### PR DESCRIPTION
**Pull request by mkborregaard**
Only works for the positional args, but that should cover very many cases

### CODE REVIEW

File: src/series.jl [hunk 1]
```diff
@@ -21,18 +21,25 @@ convertToAnyVector(n::Integer, plotattributes::KW) = Any[zeros(0) for i in 1:n],
...
 function convertToAnyVector(v::AMat, plotattributes::KW)
+    v = handlemissingsinmatrix(v)
     if all3D(plotattributes)
         Any[Surface(v)]
     else
         Any[v[:,i] for i in 1:size(v,2)]
     end, nothing
 end
 
+handlemissingsinmatrix(v::AMat) = v
```
Comment (daschw) [inline src/series.jl]: Why not just `handlemissings`?

Comment (mkborregaard) [reply src/series.jl]: no reason, I guess just to signal that it's a very local function just for use above. Happy to rename.

Comment (mkborregaard) [reply src/series.jl]: done

Comment (daschw) [reply src/series.jl]: Great, I would have been ok with `handlemissindsinmatrix` too though. I was just curious and wondering, if you were also planning to implement something like `handlemissingsinvector` or similar too, and in that case I would have suggested dispatching on the types.

**Pull request merged**